geth-utils/gethutil/trace.go

```diff
@@ -20,11 +20,12 @@ import (
 // while replaying a transaction in debug mode as well as transaction
 // execution status, the amount of gas used and the return value
 type ExecutionResult struct {
-	Invalid     bool           `json:"invalid"`
-	Gas         uint64         `json:"gas"`
-	Failed      bool           `json:"failed"`
-	ReturnValue string         `json:"returnValue"`
-	StructLogs  []StructLogRes `json:"structLogs"`
+	EnableSkippingInvalidTx bool           `json:"enable_skipping_invalid_tx"`
```

Comment:
Is there a benefit of knowing this in the tracer? Ideally we can leave this out of the actual tracer. I think the check you added in the `external-tracer/src/lib.rs` that takes `enable_skipping_invalid_tx` in account is an okay place to do it (but I would say don't store the `enable_skipping_invalid_tx` data in `TraceConfig`).